app/generateSMTScriptProp.ts

```diff
@@ -0,0 +1,7 @@
+interface Smt { smt2: string; propositions: Set<string>; }
```

Comment:
I wouldn't usually define a type/interface I wasn't going to export or something. In this case, it just cleaned up the function signature, so sure...